utils/model_to_sql.py

```diff
@@ -0,0 +1,489 @@
+"""
+    Import/Export CSV model to SQL database in either the unittest or public schema
+    and validate the files. 
+    
+    This program exists to create SQL csv backups to save or restore publish release points.
+    Also used to initialize the SQL db when migrating from csv model files.
+"""
+
+# pylint: disable=C0301,E1129,W1514,R0914,R0911,W0702
+
+import sys
+import os
+import datetime
+import csv
+from typing import List
+import psycopg
+from psycopg.rows import dict_row
+
+
+def main():
```

Comment:
This is a lot of code without tests, that makes me a bit nervous

Reply:
You are right. We should have some test cases for the util functions.
I had a test for model_to_sql.py when this code was in a different repo. I will try to create some tests.

Reply:
I did add some unit tests in tests/utils for the util function. That was a good idea.